templates/containerization/configmaps.yaml

```diff
@@ -0,0 +1,33 @@
+{{- define "tonic.configMapFromLiterals" }}
+{{- $top := first . }}
+{{- $map := index . 1 }}
+apiVersion: v1
+data:
+{{- range $literal := $map.literals }}
+    "{{ $literal.name }}": "{{ $literal.value }}"
+{{- end }}
+kind: ConfigMap
+metadata:
+  name: {{ $map.name }}
+  namespace: {{ $top.Release.Namespace }}
+  annotations:
+    {{- include "tonic.annotations" (list $top) | nindent 4 }}
+  labels:
+    {{- include "tonic.allLabels" (list $top) | nindent 4 }}
+{{- end }}
```

Comment:
should/can this function definition be nested inside the function below? (not a concern more of a question on helm idioms and whether or not that's even possible)